### DISCUSSION TITLE
How to select a hicking trail/route to see course/shape/info of it ?

### DISCUSSION THREAD

**Discussion by rein68** [osmandapp/OsmAnd#14533] [Q&A]
Hi,

I can found some trails/routes in the map, but sometimes it is not that easy to follow the whole course.
In this situation I just want to select that stuff by just one touch/click to see the whole trail highlighted.
Would be also nice to see more info about then... ;-)
But it seems not possible to do so, right?
Or did I missed something how to do so?

Greetings :-)
Reinhard

**Comment by scaidermern** [comment]
This is possible with OsmAnd 4.2. There you can tap on hiking/cycling routes to select them. Do you already have the newest version?

**Reply by rein68** [reply]
Thanks for the info! 
But in play store unfortunately only version 4.1.11 is available.

**Reply by tommycrock** [reply]
It's worth saying you have to click on the label, not just the blue line, and the label isn't necessarily on screen. 

Also it doesn't seem to work across map regions? [edit] I think it does when all the maps have the same date.

**Reply by rein68** [reply]
now finally I have version 4.2 installed and it seems to work fine, although sometimes I have to click 2 or 3 times on the label to get the track selected, or the "What's here" list is displayed showing all available routes.
I've tried this also with "OSM Public Tracks" as map source, but this seems not to work because of lack of labels?
